File: 00. Estructura/01. Lenguajes de proposito general/02. Programación básica/07. Funciones/05. Con paramatros y con retorno simple/Python/Con paramatros y con retorno simple.py

# Creación con parametros limitados
def funcion(var1, var2, varN) 
    # Contenido
    return valor

# Creación con parametros ilimitados
def funcion(*variables) 
    # Contenido
    return valor

# Creación con parametros por defecto (Los valores por defecto de pueden omitir en el llamado)
def funcion(var1 = def1, var2 = def2, varN = defN) 
    # Contenido
    return valor

# Llamado
''' Contexto ''' funcion(valor1, valor2, valorN) # Contexto

# Llamado en asignación
variable = funcion(valor1, valor2, valorN)

# Eliminación
del funcion

File: 00. Estructura/01. Lenguajes de proposito general/02. Programación básica/07. Funciones/05. Con paramatros y con retorno simple/JavaScript/Con paramatros y con retorno simple.js

// Creación con parametros limitados
function funcion(var1, var2, varN) {
    // Contenido
    return valor;
}

// Creación con parametros ilimitados
function funcion(...variables) {
    // Contenido
    return valor;
}

// Creación con parametros por defecto (Los valores por defecto de pueden omitir en el llamado)
function funcion(var1 = val1, var2 = val2, varN = valN) {
    // Contenido
    return valor;
}

// Llamado
/*Contexto*/ funcion(valor1, valor2, valorN) /*Contexto*/

// Llamado en asignación
variable = funcion(valor1, valor2, valorN);

// Eliminación
NO SOPORTADO (EL GARBAGE COLLECTOR LA ELIMINA AUTOMATICAMENTE CUANDO YA ES INACCESIBLE)

